### PR TITLE
Add ZTE Kis3 / ZTE Open C

### PR DESCRIPTION
**Pull request by ethanrjones97**
Hello! :wave: 
This PR adds support for the ZTE Open C, a slightly different version of the ZTE Kis 3\* that runs **Firefox OS**.  (That is if it wasn't a defunct project :roll_eyes: )
Currently there's no display output, but it does boot to a shell over ssh (very slow boot, be patient!) 
\*I stuck with the ZTE Kis 3 name as most of the preivous work by @KonstaT uses the same naming conventions.

### CODE REVIEW

File: aports/device/device-zte-kis3/deviceinfo [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+deviceinfo_dtb=""
+deviceinfo_modules_initfs=""
+deviceinfo_external_disk_install="true"
+deviceinfo_arch="armhf"
+
+# Device related
+deviceinfo_keyboard="false"
+deviceinfo_external_disk="true"
+deviceinfo_screen_width="800"
+deviceinfo_screen_height="480"
```
Comment (drebrez) [inline aports/device/device-zte-kis3/deviceinfo]: I think you have to switch the values:
```
deviceinfo_screen_width="480"
deviceinfo_screen_height="800"
```

File: aports/device/linux-zte-kis3/APKBUILD [hunk 2]
```diff
@@ -0,0 +1,94 @@
+# Kernel config based on: arch/arm/configs/(CHANGEME!)
```
Comment (drebrez) [inline aports/device/linux-zte-kis3/APKBUILD]: Please replace the `(CHANGEME!)` with the name of the config file you used

**Comment by ethanrjones97** [comment]
Thanks for checking out the patch @drebrez !
Yeah, that was a dumb typo :man_facepalming:  
WIll edit that file as well. 
I'm also pretty sure Travis is going to fail. The chroot lacks `xz`, when I was developing I had just installed it manually. Is there a way to add it automatically?

**Comment by drebrez** [comment]
@ethanrjones97 you can add it to the `makedepends` variable of the `linux-zte-kis3/APKBUILD` file

**Comment by ethanrjones97** [comment]
@drebrez After fighting with git I think it's in a functional state.

**Comment by ethanrjones97** [comment]
Wiki updated :heavy_check_mark:

**Pull request merged**